data/tooltips.txt

```diff
@@ -497,6 +497,12 @@ tip "shield generation:"
 tip "shield generation multiplier:"
 	`Modifies the amount of shield points recharged per second by the given factor.`
 
+tip "full shield permeability:"
+	`This percentage of all incoming damage bypasses shields and applies directly to the hull when at full shields.`
+
+tip "low shield permeability:"
+	`This percentage of all incoming damage will bypass shields and apply directly to the hull as shields approach 0 (in practice, this number is never reached, because when shields are 0 all damage goes to the hull).`
```

Comment:
At 1 hp in practice you still wouldn't see the stated value, because a single hit from anything but the most pathetic beam weapon will only be dealing a small percentage of its shield damage, and will deal most of the rest to hull. 
Shield permeability (like piercing) only has an effect if you take a single instance of shield damage that is less than your current shields, so whatever the cutoff point was you would only actually observe the stated shield permeability if you took a single instance of damage that was less than your stated shields.

Currently you can't amplify damage with this, because the total of permeability + piercing is clamped to 1.
What you can do is move one end of that linear function outside the range 0-1, which means the function reaches its maximum at some point between 0% shields and 100% shields. 
If you want to amplify damage, what you can do is combine `shield permeability` with shield and hull protections (which can be positive or negative), which actually already gives you a huge amount of room for playing around with the resulting graph of incoming shield and hull damages against shield percentage.

What I could quite easily do is also clamp that function to a non-negative result, which would also let you move one end of the function downwards, allowing you to create a steeper graph which wouldn't need to intersect any of the corners of the region - for example, shields which were 100% effective down to 60% shields, but then rapidly lost effectiveness to become 0% effective at 40% shields.

Clarifying the bounds of the values is certainly a good idea, but that should be done on the Wiki, not in the in-game tooltip.

Reply:
Being able to amplify damage depending on the shield strength sounds complex tbh (and separate from this)
We could jusg give the ship less shields and it'd work +- the same way.